api/handlers.go

```diff
@@ -26,3 +32,94 @@ func NewServer(ctx context.Context, storage interfaces.StorageFetcher, config co
 	logger := belogger.FromContext(ctx)
 	return &Server{storage: storage, logger: logger, config: config}
 }
+
+func (s Server) ObjectLifeline(ctx echo.Context, objectReference string, params ObjectLifelineParams) error {
+	limit, offset, err := checkLimitOffset(params.Limit, params.Offset)
+	if err != nil {
+		return ctx.JSON(http.StatusBadRequest, NewSingleMessageError(err.Error()))
+	}
+
+	ref, errMsg := checkReference(objectReference)
+	if errMsg != nil {
+		return ctx.JSON(http.StatusBadRequest, *errMsg)
+	}
+
+	sort := "desc"
+	if params.Sort != nil {
+		s := *params.Sort
+		if s != "desc" && s != "asc" {
```

Comment:
that always returns false. have to use OR

Reply:
no, if `s=wrong_sort`, it will be true: `s != "desc"(true) && s != "asc"(true)` -> `true`